weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -328,6 +338,16 @@ MyApplet.prototype = {
    }
 
   // Override Methods: Applet
+, _onKeySettingsUpdated: function _onKeySettingsUpdated() {
+    if (this.keybinding != null)
+      Main.keybindingManager.addHotKey(UUID,
+                                       this.keybinding,
+                                       Lang.bind(this,
+                                                 this.on_applet_clicked))
+
+    this.on_applet_clicked
```

Comment:
It wasn't; I'm not entirely certain how `bindProperty()` works but the applet worked as so when I tried it out on my machine. I just tried it again with parentheses and it, likewise, worked as it was intended so I can definitely add them in, if that's preferred.